addons/main/weapons/CfgWeapMP5.hpp

```diff
@@ -53,24 +53,28 @@ class hlc_smg_mp5N_tac: hlc_smg_MP5N {
     };
 };
 class hlc_smg_mp5k_PDW: hlc_MP5_base {
+    ACE_barrelLength = 148;
```

Comment:
https://hk-usa.com/hk-models/mp5k-pdw/ lists it as 140mm. If you are certain it's meant to be 148, can you send me your sources?

```suggestion
    ACE_barrelLength = 140;
```